src/components/CurrentWeather.vue

```diff
@@ -1,46 +1,39 @@
 <template>
   <div class="current-weather">
-    <p
-      class="current-weather__datetime"
-    >Останнє оновлення: {{this.$store.state.currentWeather.dateTime}}</p>
+    <p class="current-weather__datetime">Останнє оновлення: {{currentWeather.dateTime}}</p>
     <h2 class="current-weather__header">
-      {{this.$store.state.currentWeather.place}},
-      {{this.$store.state.currentWeather.countryCode}}
+      {{currentWeather.place}},
+      {{currentWeather.countryCode}}
     </h2>
     <div class="current-weather__columns-container">
       <div class="current-weather__left-column column">
         <h2 class="current-weather__temperature">
-          <span
-            class="current-weather__temperature-digit"
-          >{{this.$store.state.currentWeather.temperature}}</span>
+          <span class="current-weather__temperature-digit">{{currentWeather.temperature}}</span>
           °C
         </h2>
-        <p class="current-weather__description">{{this.$store.state.currentWeather.description}}</p>
+        <p class="current-weather__description">{{currentWeather.description}}</p>
       </div>
       <div class="current-weather__middle-column column">
-        <img alt="weather icon" :src="this.$store.state.currentWeather.icon">
+        <img alt="weather icon" :src="currentWeather.icon">
       </div>
       <div class="current-weather__right-column column">
         <div class="current-weather__wind-container">
           <div class="current-weather__wind-caption">вітер:</div>
           <div
-            v-if="this.$store.state.currentWeather.windSpeed"
+            v-if="currentWeather.windSpeed"
             :style="{transform:
-            'rotate('+ this.$store.state.currentWeather.windDirectionDeg+'deg)'}"
+            'rotate('+ currentWeather.windDirectionDeg+'deg)'}"
```

Comment:
you can use ES6 inside templates:
```vue
<div
:style="{
  transform: `rotate(${currentWeather.windDirectionDeg}deg)`
}"
```
As alternative you can create computed property for that